services/swarmhub/src/swarmhub/api/gridtemplate.go

```diff
@@ -0,0 +1,120 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/att-cloudnative-labs/swarmhub/services/swarmhub/src/swarmhub/db"
+	"net/http"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+func CreateGridTemplate(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	var gridTemplate db.GridTemplate
+
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&gridTemplate)
+	if err != nil {
+		message := fmt.Sprintf("invalid grid template data")
+		http.Error(w, message, http.StatusBadRequest)
+		return
+	}
+
+	gridTemplate, err = db.CreateGridTemplate(gridTemplate)
+	if err != nil {
+		message := fmt.Sprintf("error creating grid template: " + err.Error())
+		http.Error(w, message, http.StatusInternalServerError)
+		return
+	}
+
+	json, err := json.Marshal(gridTemplate)
```

Comment:
Can you rename the Marshal response from "json" to "b" or "jsonData"? Shadowing the json package could lead to confusion down the line